packages/css/src/types.ts

```diff
@@ -465,24 +464,37 @@ export interface VariantProperty {
    *
    * @see https://styled-system.com/variants
    */
-  variant: string
+  variant?: string
 }
 
-export interface UseThemeFunction {
-  (theme: any): Exclude<ThemeUIStyleObject, UseThemeFunction>
+export interface ThemeDerivedStyles {
```

Comment:
"use" is already a loaded word in React world. Tell me what you think about the rename.
Another name I considered is `ComputedStyle`.

Reply:
Emotion calls it `FunctionInterpolation`, but since we don't do CSS syntax in tagged template literals here, I'm not sure it would makes sense.